Vagrantfile

```diff
@@ -1,7 +1,11 @@
-
 Vagrant.configure("2") do |config|
   config.vm.box = "ubuntu/xenial64"
+  config.vm.provision :shell, path: "setup.sh", :privileged => false 
+  #config.vm.network :public_network, ip: "192.168.50.6"
```

Comment:
please remove unnecessary comments.

Reply:
Done. Please see latest commit.

Reply:
@DudiChen A shorter convention of accepting a comment review is to use one of the following:
ack (for acknowledge) or +1
Both terms are widely used by the open source community